package.json

```diff
@@ -134,16 +134,18 @@
         "@types/socket.io": "^1.4.32",
         "@types/socket.io-client": "^1.4.32",
         "@types/vscode": "1.40.0",
+        "@typescript-eslint/eslint-plugin": "^4.14.2",
+        "@typescript-eslint/parser": "^4.14.2",
         "copy-webpack-plugin": "^5.1.1",
+        "eslint": "^7.19.0",
+        "eslint-plugin-import": "^2.22.1",
```

Comment:
I'm a bit torn, but I think I'd rather not switch to eslint in this repo because the extension is deprecated. We don't do regular testing on this extension and the only reason we'd ever release again is for some security update or maaybe a major bug. I think we should treat the "main" branch as if it were a "rel" branch, and only push minimal changes.

Reply:
That makes sense 👍